Core.Tests/StateManagerTests.cs

```diff
@@ -1,10 +1,48 @@
-using Xunit;
+using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 
 namespace Core.Tests
 {
+	[TestFixture]
 	public class StateManagerTests
 	{
-		[Fact]
-		public void ReturnsTrue() => Assert.True(true);
+		const string t1File = "test/t1.json";
+		List<Transaction> t1 = new List<Transaction>();
+
+		[Test]
+		[Order(0)]
+		public void Import()
+		{
+			var t1Json = File.ReadAllText(t1File);
+			t1 = StateHelper.ParseTransactions(t1Json).ToList();
+			t1.Should().HaveCount(2);
+			//t1[0].CardNumber.Should().NotBe(t1[1].Hash);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6081671)